docs/Release-Process.md

```diff
@@ -28,10 +28,20 @@ Bug fixes are released as part of `MINOR` or `MAJOR` releases only. `PATCH` is a
 
 ## Release Process
 
-### Bump the `VERSION` file
+### Bump the `VERSION` and `Chart.yaml` file
 
 When the `kata-containers/kata-containers` repository is ready for a new release,
-first create a PR to set the release in the `VERSION` file and have it merged.
+first create a PR to set the release in the [`VERSION`](./../VERSION) file and update the
+`version` and `appVersion` in the
+[`Chart.yaml`](./../tools/packaging/kata-deploy/helm-chart/kata-deploy/Chart.yaml) file and
+have it merged.
+
+### Lock the `main` branch
+
+In order to prevent any PRs accidentally getting merged during the release process, we
```

Comment:
Should we explain why PRs getting merged during release is a problem?

Reply:
In theory I'm not against PRs going it as if they pass the tests they should be ready to release, but it does interrupt the caches, so I mentioned the release slowing.